packages/server/src/integrations/base/sql.ts

```diff
@@ -89,6 +89,24 @@ function parseFilters(filters: SearchFilters | undefined): SearchFilters {
   return filters
 }
 
+function generateSelectStatement(json: QueryJson, knex: Knex): any[] {
+  const { resource, meta } = json
+  const schema = meta?.table?.schema
+  return resource.fields.map(field => {
+    const shortFieldName = field.match(/(?<=\.).*$/g)?.[0]
```

Comment:
Would be good to move this regex to a global constant at the top of the file, something like 
```
const PURPOSE_REGEX = /(?<=\.).*$/g
```
I'll be honest I'm not quite sure what the purpose of the regex is here and this makes it obvious for anyone coming and looking at it, since regex is always shrouded in mystery!

Reply:
I was able to remove that regex statement altogether!
The split on the '.' was enough to get both the tableName and columnName.